JS/index.js

```diff
@@ -0,0 +1,184 @@
+const hamburger = document.getElementById('hamburger');
+const desktopList = document.getElementById('desktopList');
+const closeX = document.getElementById('closeX');
+const speakersDiv = document.getElementById('speakersDiv');
+const speakersArr = [
+  {
+    id: 1,
+    image: '../img/person1.jpg',
+    speakerName: 'Paul Godson',
+    aboutSpeaker: 'digital marketing expert',
+    speakerqualif:
+      'Paul Godson studies e-commerce from Opago Int. elite space and has 10years experience.',
+  },
+  {
+    id: 2,
+    image: '../img/person4.jpg',
+    speakerName: 'Samuel Agene',
+    aboutSpeaker: 'e-commerce lord with Misoff wellness',
+    speakerqualif:
+      'Speaker studies e-commerce from misoff wellness and an authority in the field of high ticket e-goods',
+  },
+  {
+    id: 3,
+    image: '../img/person3.jpg',
+    speakerName: 'Nurudeen Salifu',
+    aboutSpeaker: 'A graduate of Noors ecom space',
+    speakerqualif:
+      'Speaker has 18yaers experience in ecommerce with four awards from four different e-com academies',
+  },
+  {
+    id: 4,
+    image: '../img/person3.jpg',
+    speakerName: 'Nurudeen Salifu',
+    aboutSpeaker: 'A graduate of Noors ecom space',
+    speakerqualif:
+      'Speaker has 18yaers experience in ecommerce with four awards from four different e-com academies',
+  },
+  {
+    id: 5,
+    image: '../img/person3.jpg',
+    speakerName: 'Nurudeen Salifu',
+    aboutSpeaker: 'A graduate of Noors ecom space',
+    speakerqualif:
+      'Speaker has 18yaers experience in ecommerce with four awards from four different e-com academies',
+  },
+  {
+    id: 6,
+    image: '../img/person3.jpg',
+    speakerName: 'Nurudeen Salifu',
+    aboutSpeaker: 'A graduate of Noors ecom space',
+    speakerqualif:
+      'Speaker has 18yaers experience in ecommerce with four awards from four different e-com academies',
+  },
+];
+const ulLi = document.querySelectorAll('.ulLi');
+const liArr = Array.from(ulLi);
+
+hamburger.addEventListener('click', () => {
+  desktopList.classList.toggle('showUlItems');
+  closeX.classList.toggle('showUlItems');
+  hamburger.classList.toggle('hide');
+});
+
+closeX.addEventListener('click', () => {
+  hamburger.classList.toggle('hide');
+  desktopList.classList.toggle('showUlItems');
+  closeX.classList.toggle('showUlItems');
+});
+
+liArr.forEach((el) => {
+  el.addEventListener('click', () => {
+    hamburger.classList.remove('hide');
+    desktopList.classList.remove('showUlItems');
+    closeX.classList.remove('showUlItems');
+  });
+});
+
+let data;
+function speakers() {
+  data = `<div class="d-md-flex flex-wrap gap-4">
+  <div class="allSpeakers my-4 col-md-5">
+    <div class="d-flex speaker1">
+      <img src="${speakersArr[0].image}" alt="first Speaker" />
+      <div class="personName ms-3">
+        <h4>${speakersArr[0].speakerName}</h4>
+        <p><i class="text-danger">${speakersArr[0].aboutSpeaker}</i></p>
```

Comment:
- [ ] Kindly apply `font-weight` to the `h4` element based on the design template. I would advise you to use a `font-weight` of `700`. 👍 

![image](https://user-images.githubusercontent.com/77106962/181353055-3a9ac88c-5c1d-41e0-9996-d80b7e5408ac.png)

Reply:
700 font width was applied